CONTRIBUTING.md

```diff
@@ -1,76 +1,268 @@
 * Include screenshots and animated gifs of your changes whenever possible.
 
-## Adding or Editing Site Content
+## Review Apps
 
-### Apps
-
-See [github.com/electron/electron-apps#adding-your-app](https://github.com/electron/electron-apps#adding-your-app).
+[Heroku Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+make it easier for reviewers to see the effects of a proposed change. Whenever 
+a pull request is opened on the website repo, the branch is autodeployed 
+to Heroku with a unique URL like `electron-website-pr-787.herokuapp.com`. 
+You'll see a notification on GitHub after the branch has been deployed.
+When pull request is closed or merged, the ephemeral app is destroyed.
 
-### Meetups
+![review app notification](https://user-images.githubusercontent.com/2289/31911130-8a0f91da-b7f4-11e7-9784-9ad609321c18.png)
 
-If you want to add a meetup to the community site please open a pull request.
+## Development
 
-Add the meetup to the **bottom** of the list by editing [_data/meetups.yml](/_data/meetups.yml).
+Follow these steps to copy this repository to your computer and build the site:
 
-```yml
--
-  name: Name of Group
-  location: City, State (if applicable)
-  country: Country
-  href: Link to meetup's site
+```bash
+git clone https://github.com/electron/electron.atom.io
+cd electron.atom.io
+npm install
+npm run dev
 ```
 
+You should now have an Express server running at 
+[localhost:5000](http://localhost:5000) :tada:
+
+Read on for more info about the structure of the site.
+
+- [Content](#content)
+  - [Documentation](#documentation)
+  - [Apps](#apps)
+  - [Blog](#blog)
+  - [Translations](#translations)
+- [Routes](#routes)
+- [Middlware](#middleware)
+- [Views](#views)
+- [Context](#context)
+- [Styles](#styles)
+- [Fontend JavaScript](#frontend-javascript)
+- [Scripts](#scripts)
+- [Environment Variables](#environment-variables)
+
+## Content
+
+The content on [electronjs.org] comes from a number of sources.
+
 ### Documentation
 
-The documentation on this site is pulled directly from the `electron/electron` repository's `docs` directory. Contributions to the documentation should be made there: [electron/electron](https://github.com/electron/electron/tree/master/docs).
+Electron's English API docs and tutorials live in the [electron/electron] repo 
+in the `docs` directory. Contributions to the documentation should be made there: [electron/electron].
+
+After being [translated](#translations) on Crowdin, all the docs are automatically
+updated on the [electron/electron-i18n] repo and auto-published to npm as the
+`electron-i18n` package. That package is in turne used by this website repo
+as the source of all documentation. See [electron/electron-i18n#usage](https://github.com/electron/electron-i18n#usage) for an explanation of the
+documentation data's structure.
+
+### Apps
+
+Want to add your app to the site? There's a repo for that. 
```

Comment:
Maybe have a lead in sentence here like: Apps listed on the Electron website are a collection of applications that developers have submitted to be included on the site.  If you want to add your app to the site, see [electron/electron-apps](https://github.com/electron/electron-apps#adding-your-app).

Something like that maybe?